.github/workflows/catnip.yml

```diff
@@ -46,6 +46,28 @@ jobs:
         echo -e "\tPasswordAuthentication no" >> $HOME/.ssh/config
         echo -e "\tUser ${{ secrets.USERNAME }}" >> $HOME/.ssh/config
         echo -e "\tPort ${{ secrets.PORTNUM }}" >> $HOME/.ssh/config
+    - name: Bootstrap VMs
+      shell: bash
+      run: |
+        echo "Running azure.sh on SERVER"
+        ssh $SERVER "sudo bash ~/demikernel/scripts/setup/azure.sh"
+
+        echo "Running hugepages.sh on SERVER"
+        ssh $SERVER "sudo bash ~/demikernel/scripts/setup/hugepages.sh"
+
+        echo "Running ip addr flush on SERVER"
+        ssh $SERVER "sudo ip -br addr show dev eth1 | grep -q ' UP ' && \
+                     sudo ip addr flush dev eth1"
+
+        echo "Running azure.sh on CLIENT"
+        ssh $CLIENT "sudo bash ~/demikernel/scripts/setup/azure.sh"
+
+        echo "Running hugepages.sh on CLIENT"
+        ssh $CLIENT "sudo bash ~/demikernel/scripts/setup/hugepages.sh"
+
+        echo "Running ip addr flush on CLIENT"
+        ssh $CLIENT "sudo ip -br addr show dev eth1 | grep -q ' UP ' && \
+                     sudo ip addr flush dev eth1"
```

Comment:
We may want to add these steps in the release pipeline as well. A retrigger of the pipeline for any reason, would not re-run the bootstrap steps.

Reply:
They are already added in the release-pipeline section as well. (If you scroll to Ln no. 121)

Reply:
For some reason they did not render in VS code. Sorry for signaling it wrong.